### ISSUE TITLE
Problem with fetch_tickers() with Lbank

### ISSUE THREAD

**Issue by HornKild** [ccxt/ccxt#3239]
Hello,

Everything works fine with others exchange that allow to fetch all tickers at once with a single call but for some reason it returns an error with Lbank : 
```
In [146]: ccxt.lbank().fetch_tickers()
Traceback (most recent call last):

  File "<ipython-input-146-dbb317dca597>", line 1, in <module>
    ccxt.lbank().fetch_tickers()

  File "/home/myhome/anaconda3/lib/python3.6/site-packages/ccxt/lbank.py", line 193, in fetch_tickers
    market = self.marketsById[id]

KeyError: 'ven_btc'
```
The same function with Kucoin, Bitz, etc. returns the tickers: 
`ccxt.kucoin().fetch_tickers()`

VERSION 1.14.267

What could be the reason ?

**Comment by kroitor** [comment]
Duplicate: #3179

**Comment by kroitor** [comment]
@HornKild you can send them a line as well, like other users did, to push on them a little. They might fix it quicker then.

**Comment by HornKild** [comment]
Hello,

This is a new error I have with LBank when fetching tickers or markets : 

```
lbank = ccxt.lbank()
lbank.verbose = True
lbank.fetch_tickers()

Request: GET https://api.lbank.info/v1/currencyPairs.do {'User-Agent': 'python-requests/2.18.4', 'Accept-Encoding': 'gzip, deflate'} None

Response: GET https://api.lbank.info/v1/currencyPairs.do 200 {'Server': 'Tengine', 'Date': 'Fri, 06 Jul 2018 20:59:04 GMT', 'Content-Type': 'text/plain;charset=ISO-8859-1', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'X-Frame-Options': 'SAMEORIGIN', 'Content-Encoding': 'gzip'} ["pch_usdt","dax_usdt","btc_usdt","eth_usdt","bch_usdt","neo_usdt","vet_erc20_usdt","qtum_usdt","zec_usdt","pai_btc","pch_btc","pnt_btc","dax_btc","eth_btc","bch_btc","ltc_btc","neo_btc","dash_btc","vet_erc20_btc","etc_btc","qtum_btc","btg_btc","zec_btc","sc_btc","bts_btc","bcd_btc","cmt_btc","abt_btc","cpx_btc","xwc_btc","sbtc_btc","fil6_btc","fil12_btc","fil36_btc","btf_btc","pai_eth","bfdt_eth","her_eth","luc_eth","ptt_eth","wicc_eth","tac_eth","qkc_eth","afc_eth","idhub_eth","sead_eth","ssc_eth","skm_eth","pch_eth","iic_eth","ply_eth","ext_eth","dax_eth","bch_eth","eos_eth","trx_eth","vet_erc20_eth","qtum_eth","zec_eth","bts_eth","btm_eth","mith_eth","nas_eth","man_eth","cmt_eth","dbc_eth","bto_eth","abt_eth","ddd_eth","cpx_eth","cs_eth","iht_eth","ink_eth","tky_eth","ocn_eth","dct_eth","yoyow_eth","zsc_eth","zpt_eth","eko_eth","mda_eth","pst_eth","xwc_eth","put_eth","tgc_eth","pnt_eth","aac_eth","fil6_eth","fil12_eth","fil36_eth","uip_eth","lch_eth","seer_eth","bsb_eth","cdc_eth","grams_eth","ddm_eth","eai_eth","inc_eth","ply_qtum","mith_qtum","ink_qtum","bot_qtum","qbt_qtum","put_qtum","ent_qtum","chat_qtum","aac_qtum","yoyow_bts","seer_bts","bnb_usdt","ht_usdt","sead_usdt"]
Traceback (most recent call last):

  File "<ipython-input-44-8a84a7af2d5c>", line 1, in <module>
    lbank.fetch_tickers()

  File "/home/xx/anaconda3/lib/python3.6/site-packages/ccxt/lbank.py", line 190, in fetch_tickers
    self.load_markets()

  File "/home/xx/anaconda3/lib/python3.6/site-packages/ccxt/base/exchange.py", line 919, in load_markets
    markets = self.fetch_markets()

  File "/home/xx/anaconda3/lib/python3.6/site-packages/ccxt/lbank.py", line 110, in fetch_markets
    baseId, quoteId = id.split('_')

ValueError: too many values to unpack (expected 2)

print(ccxt.__version__)
1.15.21
```

**Comment by kroitor** [comment]
vet_erc20_usdt...

**Comment by kroitor** [comment]
- https://www.reddit.com/r/Vechain/comments/7m76yb/is_ven_an_erc20_token/
- https://medium.com/@vechainofficial/vechain-apotheosis-the-beginning-d9f0fdbdc910

`VEN` is being rebranded to `VET`.

**Comment by HornKild** [comment]
Hi Kroitor, is it something that can be fixed ?

**Comment by kroitor** [comment]
@HornKild hi, yes, it is fixed in 1.15.24.
